daemon/execdriver/native/seccomp_unsupported.go

```diff
@@ -0,0 +1,10 @@
+// +build linux,!seccomp
+
+package native
+
+import "github.com/opencontainers/runc/libcontainer/configs"
+
+var (
+	seccompSupported      = false
+	defaultSeccompProfile = &configs.Seccomp{}
```

Comment:
could this just be `nil` and then you could probably remove the need for the boolean.  Anyways, nice job on this ;)